appveyor.yml

```diff
@@ -18,13 +20,29 @@ build_script:
   - dotnet build src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION% %STRONG_NAME_OPTIONS%
   - cmd: IF "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION%  &  dotnet pack src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION% %STRONG_NAME_OPTIONS%)
   - cmd: IF NOT "%APPVEYOR_REPO_TAG%" == "true" (dotnet pack src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION% --version-suffix ci-%APPVEYOR_BUILD_NUMBER%  &  dotnet pack src/Confluent.Kafka/Confluent.Kafka.csproj -c %CONFIGURATION% %STRONG_NAME_OPTIONS% --version-suffix ci-%APPVEYOR_BUILD_NUMBER%)
+  - docfx doc/docfx.json
+
+after_build:
+  - cmd: IF "%APPVEYOR_REPO_TAG%" == "true" (7z a docs-%VERSION%.zip doc/_site/*)
+  - cmd: IF NOT "%APPVEYOR_REPO_TAG%" == "true" (7z a docs-%VERSION%-ci-%APPVEYOR_BUILD_NUMBER%.zip doc/_site/*)
 
 test_script:
  - dotnet test -c %CONFIGURATION% --no-build test/Confluent.Kafka.UnitTests/Confluent.Kafka.UnitTests.csproj
  - dotnet test -c %CONFIGURATION% test/Confluent.Kafka.StrongName.UnitTests/Confluent.Kafka.StrongName.UnitTests.csproj
 
 artifacts:
   - path: ./src/Confluent.Kafka/bin*/Release/*.nupkg
+  - path: ./docs*.zip
+
+deploy:
+  - provider: S3
+    access_key_id: AKIAI6DI3ICJPXUPPPKA
```

Comment:
yep, I intend to switch this back (and will create new credentials obviously). i switched it over whilst trying to work out why S3 deploy wasn't working (the answer being it doesn't function on PR builds). The secure key is secret still of course, and there is no a huge security risk anyway as it only has credentials for the one bucket.